xbmc/main/win32/WinMain.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+ *      Copyright (C) 2005-2013 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "threads/platform/win/Win32Exception.h"
+#include <shellapi.h>
+#include <dbghelp.h>
+#include "threads/Thread.h"
+#include "utils/CPUInfo.h"
+#include "xbmc.h"
+#include "CompileInfo.h"
+
+
+extern "C" int main(int argc, char* argv[]);
+
+// Minidump creation function
+LONG WINAPI CreateMiniDump( EXCEPTION_POINTERS* pEp )
```

Comment:
May aswell fixup the coding style. A lot of places where there are unnecessary spaces and some (after `if` etc.) where spaces are missing.